### PR TITLE
fix: Limit speakers calls/cfs info 

### LINKED ISSUE #5998: Fetching featured speakers, accepted sessions, public call for speaker by default
**Describe the bug**
By default, the server should only return featured speakers, accepted sessions, public speakers call when client-side make GET requests. Currently, it also returns pending sessions or private call for speaker.

**Additional context**
Discussed in: https://github.com/fossasia/open-event-android/pull/1902 (comment)

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #5998 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Fetches only accepted sessions and public cfs.

#### Changes proposed in this pull request:

- Changed the queries in query methods to filter out the accepted sessions and public cfs

### CODE REVIEW

File: app/api/sessions.py [hunk 1]
```diff
@@ -86,7 +86,7 @@ def query(self, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        query_ = self.session.query(Session)
+        query_ = self.session.query(Session).filter(Session.state == 'accepted')
```
Comment (iamareebjamal) [inline app/api/sessions.py]: Organizer should be able to see all in selected places. Please check it works. Also, attendee should not be able to see pending ones even after using filtering. Please check this

**Comment by mrsaicharan1** [comment]
@uds5501 @shreyanshdwivedi Can you please verify if this isn't breaking the speakers-call page? For some reason `current_identity` is being read as `None`, even though I'm logged in.

**Comment by bhaveshAn** [comment]
@mrsaicharan1 I guess for fixing Travis, you have to update the API docs according to the http response you are getting now.

**Comment by mrsaicharan1** [comment]
Will continue with this as soon as I'm done with solving some crucial issues. This needs a lot of testing. Hence, closing this. In the meanwhile, if anybody finds a fix and can be assured that this works, after testing, please feel free to do it.

**Pull request closed without merging**